src/DefinitionGenerator.ts

```diff
@@ -1,10 +1,11 @@
-import { JSONSchema7 } from 'json-schema';
+import _ = require('lodash');
```

Comment:
this syntax looks weird. how does it work? I've never seen it before

also, we normally use ramda 🙈 But I can see how much it makes sense to just replace the existing utils with lodash here 👍

Reply:
This is an open source project, I would prefer to keep it easy for people to contribute - `lodash` is more aligned with this idea than `ramda`